types/alexa-sdk/index.d.ts

```diff
@@ -74,12 +77,22 @@ interface SessionUser {
 interface LaunchRequest extends IRequest { }
 
 interface IntentRequest extends IRequest {
+    dialogState: DialogStates;
     intent: Intent;
 }
 
+interface ISlots {
```

Comment:
Instead of this, consider a declaration for the values that these object map to (i.e. an interface potentially named `SlotValue` containing `confirmationStatus`, `name`, and `value`), and then use `Record<string, SlotValue>`.

Also, be aware that no other type in this package starts with `I`. Best to be consistent.